internal/backend/sessionsgrpcsvc/svc.go

```diff
@@ -42,20 +40,11 @@ func (s *server) Start(ctx context.Context, req *connect.Request[sessionsv1.Star
 	}
 
 	for k, v := range msg.JsonInputs {
-		d := json.NewDecoder(strings.NewReader(v))
```

Comment:
you have to decode first, otherwise it'll just convert the json into the string. this is not the problem.